crates/libs/bindgen/src/lib.rs

```diff
@@ -195,7 +196,8 @@ where
         namespace: "",
     };
 
-    writer.write(tree)
+    writer.write(tree);
+    index::write();
```

Comment:
As this is not specific or applicable to most invocations of `windows-bindgen` we should probably just have another static tool that generates the json like tool_yml and the rest.

Reply:
It's currently in bindgen to take advantage of the filters. I *think* everything else is externally accessible but I'd have to double check.